README.asciidoc

```diff
@@ -304,22 +304,52 @@
 ==== Server-specific configuration
 
 Many servers accept configuration options that are not part of the LSP spec.  The TOML table
-`[language.<filetype>.settings]` holds those configuration options.  It has the same structure
+`[language_server.<server_name>.settings]` holds those configuration options.  It has the same structure
 as the corresponding fragments from VSCode's `settings.json`. For example:
 
 [source,toml]
 ----
-[language.go]
+[language_server.gopls]
 ...
-settings_section = "gopls"
-[language.go.settings.gopls]
+[language_server.gopls.settings.gopls]
 "formatting.gofumpt" = true
 ----
 
-During server initialization, kak-lsp sends the section specified by `settings_section`; in this
-case `{"formatting.gofumpt":true}`.  Whenever you change the Kakoune option `lsp_config`, the
-same section is sent via `workspace/didChangeConfiguration`.  Additionally, kak-lsp will send
-arbitrary sections that are requested by the server in `workspace/configuration`.
+During server initialization, kak-lsp sends either the section specified by `settings_section`
+or the section with same name as the server; in this case `{"formatting.gofumpt":true}`.
+Whenever you change the Kakoune option `lsp_config`, the same section is sent via
+`workspace/didChangeConfiguration`.  Additionally, kak-lsp will send arbitrary sections that
+are requested by the server in `workspace/configuration`.
+
+==== Multiple language servers
+
+It is possible to map more than one language server to filetypes. For example, if you want to
+set up TSServer and TailwindCSS to use in React projects:
+
+[source,toml]
+----
+[language_ids]
+javascript = "javascriptreact"
+typescript = "typescriptreact"
```

Comment:
I made another comment suggesting default language ID mappings.
Let's make sure this remains in sync with the default language ID mapping in kak-lsp.toml.